server.go

```diff
@@ -2794,6 +2808,28 @@ func (s *Server) writeErrorResponse(bw *bufio.Writer, ctx *RequestCtx, serverNam
 	return bw
 }
 
+func (s *Server) trackConn(c net.Conn, state ConnState) {
+	switch state {
+	case StateIdle:
+		if s.idleConns == nil {
+			s.idleConns = make(map[net.Conn]struct{})
+		}
+		s.idleConns[c] = struct{}{}
+
+	default:
+		delete(s.idleConns, c)
+	}
+}
+
+func (s *Server) closeIdleConns() {
+	s.connsMu.Lock()
+	for c := range s.idleConns {
+		c.Close()
+		s.setStateLocked(c, StateClosed)
```

Comment:
This is going to call `s.ConnState` which shouldn't happen here as it's already happening here: https://github.com/valyala/fasthttp/blob/2a6f7db5bbc4d7c11f1ccc0cb827e145b9b7d7ea/workerpool.go#L237

Reply:
Oh, I missed it. 
Updated.